mifosng-provider/src/main/java/org/mifosplatform/organisation/staff/serialization/StaffCommandFromApiJsonDeserializer.java

```diff
@@ -134,6 +148,17 @@ public void validateForUpdate(final String json) {
 
         if (this.fromApiJsonHelper.parameterExists("isActive", element)) {
             final Boolean activeFlag = this.fromApiJsonHelper.extractBooleanNamed("isActive", element);
+            //Need to add here check to see if any clients and loans are assigned to this staff if staff is being set to inactive --LJB
+            final Boolean forceStatus = this.fromApiJsonHelper.extractBooleanNamed("forceStatus", element);
+            if ((activeFlag == false && forceStatus == null) || 
+                (activeFlag == false && forceStatus != true)) {
+                List<ClientData> clientData = (List<ClientData>) clientReadPlatformService.retrieveAllForLookup("staff_id = "+ staffId);
```

Comment:
Can you please add validate for group, loan, savings too. And also please consider the status(not in closed state) of these entities(loan, savings, client and group) while validating